File: app/src/main/java/coolpharaoh/tee/speicher/tea/timer/views/export_import/datatransfer/pojo/StatisticsPOJO.java
package coolpharaoh.tee.speicher.tea.timer.views.export_import.datatransfer.pojo;

import androidx.room.ColumnInfo;

public class StatisticsPOJO {
    @ColumnInfo(name = "name")
    public String teaname;

    @ColumnInfo(name = "color")
    public int teacolor;

    @ColumnInfo(name = "counter")
    public long counter;
}

File: app/src/main/java/coolpharaoh/tee/speicher/tea/timer/views/export_import/data_transform/pojo/StatisticsPOJO.kt
package coolpharaoh.tee.speicher.tea.timer.views.export_import.data_transform.pojo

import androidx.room.ColumnInfo

class StatisticsPOJO {
    @JvmField
    @ColumnInfo(name = "name")
    var teaName: String? = null

    @JvmField
    @ColumnInfo(name = "color")
    var teaColor: Int = 0

    @JvmField
    @ColumnInfo(name = "counter")
    var counter: Long = 0
}